workshop/exercise-6/README.md

```diff
@@ -116,3 +116,11 @@ Depending on whether a service handles [HTTP](https://istio.io/docs/reference/co
 * [Traffic Management](https://blog.openshift.com/istio-traffic-management-diving-deeper/)
 * [Circuit Breaking](https://blog.openshift.com/microservices-patterns-envoy-part-i/)
 * [Timeouts and Retries](https://blog.openshift.com/microservices-patterns-envoy-proxy-part-ii-timeouts-retries/)
+
+
+
+## Questions
+1. Where are routing rules defined?  Options: (VirtualService, DestinationRule, ExternalService)  Answer: VirtualService
+1. Where are service versions (subsets) defined?  Options: (VirtualService, DestinationRule, ExternalService)  Answer: DestinationRule
+1. Which Istio component is responsible for sending traffic management configurations to Istio sidecars?  Options: (Mixer, Citadel, Pilot, Kubernetes)  Answer: Pilot
+1. What is the name of the proxy that runs in Istio sidecars and routes requests within the service mesh?  Options: (NGINX, Envoy, HAProxy)  Answer: Envoy
```

Comment:
maybe use "default proxy" instead of "proxy"?   Given you could use nginx to work with Istio.